US pin-home06 documentation (#162)
exoplatform/exo-documentation: docs/Administration.rst

   To invoke a method of the selected service, click Run corresponding
   to it. After invoking it successfully, you will see the result
   returned if any.
   
   
.. _ManagingNews:

==============
Managing News
==============

> How to manage pinned block layouts

Reviewer: ```suggestion
How to display pinned News on a page
```

-----------------------------------

The home page CLV is a block that displays articles that have been pinned on the homepage by a user having the role publisher.

> The layout is not set by default, you have to edit the page layout then add a "Content by query" in the area in which you want to set the block.

Reviewer: I would replace this paragraph by something like:
"The pinned articles are not highlighted by default in the platform. In order to display them in a page, edit the page layout then add a "Content by query" application at the desired location."
And set a link on "pinned articles" to the user guide explaining how to pin an article.

Author: updated

|image348| 

|image349| 

> The first thing to do is to edit the portlet and choose the folder path.

Reviewer: ```suggestion
Once the application is added in the page, edit its settings and choose the folder path.
```

|image350| 

Then you have to choose the appriopriate template from "Display Settings" tab of "Edit Mode" : 

> - For the blog layout, select the following file "NewsPin.gtmpl".

Reviewer: ```suggestion
- For the blog layout, select "NewsPin.gtmpl".
```

> - For the latest News layout, select the following file "LatestNews.gtmpl". 

Reviewer: ```suggestion
- For the latest News layout, select "LatestNews.gtmpl". 
```

|image351| 

> Next, fill in the field "Header", this information will be displayed in the top right of the block.

Reviewer: top left ?

|image352| 

From advanced tab, fill in the following query in the field "By query" :

       *select * from exo:symlink where jcr:path like '/Application Data/News/Pinned/%' ORDER BY jcr:created DESC*

> Then select the appropriate page from News folder from the selector "Show in page"

Reviewer: ```suggestion
Then select the page *News* > *Details* from the selector "Show in page"
```
